README.md

```diff
@@ -3,6 +3,11 @@ This is an app for OS X that can (re)sign apps and bundle them into ipa files th
 
 Supported input types are: ipa, deb, app, xcarchive
 
+Changes made in this fork
+------
+The upload artifact version and the MacOS development target versions have been changed in order to have a successful autobuild. 
+A pull request has been made to the original project but as of 2/11/24 it has not been merged and therefore this is the most up-to-date version of iOS App Signer with successful builds for MacOS 15.0+
+
```

Comment:
please drop this from the PR

Reply:
Sorry about that, the PR should be ready now